lib/components/src/Button/Button.tsx

```diff
@@ -83,14 +83,12 @@ const ButtonWrapper = styled.button<ButtonWrapperProps>(
   ({ theme, primary, secondary, gray }) => {
     let color;
 
-    if (primary) {
-      color = theme.color.primary;
-    }
-    if (secondary) {
-      color = theme.color.secondary;
-    }
     if (gray) {
       color = theme.color.medium;
+    } else if (secondary) {
+      color = theme.color.secondary;
+    } else if (primary) {
+      color = theme.color.primary;
```

Comment:
is there a reason why secondary is above primary?

Reply:
Make the assignment more explicit. Without reassign on next line.